src/lxc/lxccontainer.c

```diff
@@ -4659,6 +4680,152 @@ static bool do_lxcapi_restore(struct lxc_container *c, char *directory, bool ver
 
 WRAP_API_2(bool, lxcapi_restore, char *, bool)
 
+static int do_lxcapi_mount(struct lxc_container *c,
+			const char *source, const char *target,
+			const char *filesystemtype, unsigned long mountflags,
+			const void *data, struct lxc_mount *mnt) {
+	int ret = -1, len, fd, status;
```

Comment:
Use `size_t` instead of `int` for `len` since it is defined as the datatype to use when querying the size of an object. Also `malloc()` and friends expect `size_t`.